microsoft/restler-fuzzer
# Enable traceDB-based replay without the request text

For use cases when a trace database is saved and re-played, this change allows using only replay blocks without the rendered requests. This is being added for scenarios where only replay blocks are saved during custom serialization.

The rendered requests continue to be logged by default, since they are useful for debugging.

Testing:
- manual testing
- adding logic to the database replay unit test to create a database without the request text and confirm replay matches the version with the request text.
